src/v/cloud_storage/remote_partition.cc

```diff
@@ -890,11 +889,10 @@ remote_partition::finalize(ss::abort_source& as) {
 
     if (manifest_get_result != download_result::success) {
         vlog(
-          _ctxlog.debug,
-          "Failed to fetch manifest during finalize(), maybe another node "
-          "already completed deletion. Error: {}",
+          _ctxlog.error,
```

Comment:
Hmm, maybe this should remain a WARN until we put something in the scrubber to wait for a grace period before cleaning up?  idk, it'll mainly be an issue in tests I suppose, and you've already added log allow lists.